html/semantics/document-metadata/interactions-of-styling-and-scripting/stylesheet.py

```diff
@@ -0,0 +1,10 @@
+from time import sleep
+def main(request, response):
+  if "delay" in request.GET:
+    delay = int(request.GET["delay"])
+    sleep(delay)
+
+  if "stylesNotMatchingEnvironment" in request.GET:
+    return 'h1 {color: green}'
+  else:
+    return 'h1 {color: red}'
```

Comment:
Usually green is pass and red is fail.

Reply:
Ok I think I'll just make these random colors or something because I'm not trying to indicate pass/fail here. Obviously if the "stylesNotMatchingEnvironment" styles were applied over the other styles (when the environment doesn't match), the implementation fails a lot more fundamental things than this test is testing.